Jenkinsfile

```diff
@@ -31,9 +32,36 @@ pipeline {
           def image = docker.build "lowkent2me/bank:${env.GIT_COMMIT}"
           docker.withRegistry('','dockerhub-kvs') {
             image.push()
+          build = "${env.GIT_COMMIT}"
+          }
+        }
+      }
+    }
+    stage('Update Helm Chart') {
+      when { expression { build == "${env.GIT_COMMIT}" } }
```

Comment:
lets make a logical and here that checks that branch is master and expression is  build == "${env.GIT_COMMIT}"